Fix the annoying menu/tooltip 'punch-through' problem in -multiwindow mode.

---
 xserver/hw/xwin/winmultiwindowwndproc.c |    1 +
 1 file changed, 1 insertion(+)

Index: xorg-server-1.5.3/xserver/hw/xwin/winmultiwindowwndproc.c
===================================================================
--- xorg-server-1.5.3.orig/xserver/hw/xwin/winmultiwindowwndproc.c	2009-02-16 12:34:19.953125000 +0000
+++ xorg-server-1.5.3/xserver/hw/xwin/winmultiwindowwndproc.c	2009-02-16 12:36:40.093750000 +0000
@@ -981,6 +981,7 @@
 #endif
       /* Adjust the X Window to the moved Windows window */
       winAdjustXWindow (pWin, hwnd);
+      if (wParam == SIZE_MINIMIZED) winReorderWindowsMultiWindow();
       return 0; /* end of WM_SIZE handler */
 
     case WM_MOUSEACTIVATE:
